### PR TITLE
Adds syntax bug report issue template

### PR DESCRIPTION
**Pull request by LuqueDaniel**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/syntax_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Syntax bug report
+description: Report a bug related to syntax
+labels: ["topic:syntax"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for the report. Please follow the steps below:
+        - Read the [contributing guide](https://github.com/LuqueDaniel/vscode-language-renpy/blob/master/Contributing.md#reporting-bugs-or-proposing-features)
+        - Write a descriptibe title for your issue.
```
Comment (JelleInfinity) [inline .github/ISSUE_TEMPLATE/syntax_bug_report.yml:10]: You've made a spelling mistake there 'descriptive'

File: .github/ISSUE_TEMPLATE/syntax_bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      placeholder: 2.1.0, 2.0.11...
+    validations:
+      required: true
+
+  - type: textarea
+    id: issue-description
+    attributes:
+      label: Issue description
+      description: |
+        Describe the issue. What doesn't work? and how do you expect it to work instead?
```
Comment (JelleInfinity) [inline .github/ISSUE_TEMPLATE/syntax_bug_report.yml:27]: Probably want capital A there

File: .github/ISSUE_TEMPLATE/syntax_bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        You can include images with drag and drop or pasting them.
+    validations:
+      required: true
+
+  - type: textarea
+    id: scope-inspector-screenshots
+    attributes:
+      label: Scope inspector screenshots
+      description: |
+        Please provide screenshots with the [scope inspector](https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#scope-inspector).
```
Comment (JelleInfinity) [inline .github/ISSUE_TEMPLATE/syntax_bug_report.yml:37]: Could you make this;

Please provide screenshots of the syntax you believe is incorrectly highlighted. Include screenshots of the scopes on and around the incorrect area using the [scope inspector]

**Review by JelleInfinity — CHANGES_REQUESTED** [review]
Awesome :) Just a minor change request

File: .github/ISSUE_TEMPLATE/syntax_bug_report.yml (hunk 1 above)
Comment (LuqueDaniel) [reply .github/ISSUE_TEMPLATE/syntax_bug_report.yml]: 😅

File: .github/ISSUE_TEMPLATE/syntax_bug_report.yml (hunk 3 above)
Comment (LuqueDaniel) [reply .github/ISSUE_TEMPLATE/syntax_bug_report.yml]: Ok!

**Review by JelleInfinity — APPROVED** [review]
Looks great :)

**Pull request merged**